### ISSUE TITLE
rxExecBy in Spark hangs if keys is a named vector

### ISSUE THREAD

**Issue by hongooi73** [RevolutionAnalytics/dplyrXdf#46]
```r
mthc <- dplyrXdf::copy_to_hdfs(mtcars)

# works
rxExecBy(mthc, "cyl", function(keys, data) 0)

# hangs
x <- c(cyl="cyl")
rxExecBy(mthc, x, function(keys, data) 0)
```

**Comment by hongooi73** [comment]
TFS item 78055

**Comment by hongooi73** [comment]
fixed in 9.2
